find.py

```diff
@@ -0,0 +1,52 @@
+def find(a,b,n):
+    low_a=0
+    low_b=0
+    high_a=len(a)-1
+    high_b=len(b)-1
+
+    while low_a<=high_a and low_b<=high_b:
+        if a[low_a]+b[low_b]>n or a[high_a]+b[high_b]<n:
+            return (-1,-1)
+        elif a[low_a]+b[high_b]==n:
+            return (low_a,high_b)
+        elif a[low_a]+b[high_b]<n:
+            lower_a=low_a

```

Comment:
define a function to get a better low_a and call the function here, to make your code clearer.